plugins/woocommerce/includes/admin/meta-boxes/views/html-order-item.php

```diff
@@ -13,14 +13,15 @@
 $product_link = $product ? admin_url( 'post.php?post=' . $item->get_product_id() . '&action=edit' ) : '';
 $thumbnail    = $product ? apply_filters( 'woocommerce_admin_order_item_thumbnail', $product->get_image( 'thumbnail', array( 'title' => '' ), false ), $item_id, $item ) : '';
 $row_class    = apply_filters( 'woocommerce_admin_html_order_item_class', ! empty( $class ) ? $class : '', $item, $order );
+$item_name    = apply_filters( 'woocommerce_order_item_name', $item->get_name(), $item, $is_visible );
```

Comment:
`$is_visible` does not seem to be defined here -- you probably need to add that:

`$is_visible = $product && $product->is_visible();`